test/k8s-integration/main.go

```diff
@@ -59,6 +61,7 @@ var (
 const (
 	pdImagePlaceholder = "gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver"
 	k8sBuildBinDir     = "_output/dockerized/bin/linux/amd64"
+	gkeTestClusterName = "gcp-pd-csi-driver-test-cluster"
```

Comment:
Can we append some random suffix to it? That will let us run multiple test instances in parallel.  Or if some previous test cluster failed to cleanup properly, then subsequent test runs would not get blocked.

Reply:
Unless I'm mistaken, the current kube-up script does not do this either. Are separate tests run through prow supposed to be run on separate projects, thus avoiding any concurrent and subsequent conflicts.

Reply:
Yes, technically with boskos project lending, there shouldn't be concurrent runs in the same project.  However, I'm thinking more about the scenario where a previous run failed to properly clean up the cluster (for whatever reason).

The kube-up script first tries to delete any existing cluster before trying to create a new one to get around this issue.  We may want to consider doing something similar for gke.